cla_backend/settings/base.py

```diff
@@ -463,8 +463,9 @@ def bank_holidays_cache_adapter_factory():
 
 EMAIL_ORCHESTRATOR_URL = os.environ.get("EMAIL_ORCHESTRATOR_URL")
 
+# This default will be picked up by running tests in an IDE which does not set CFE_HOST
 CFE_URL = (
-    os.environ.get("CFE_HOST", "https://cfe-civil-staging.cloud-platform.service.justice.gov.uk") + "/v6/assessments"
+    os.environ.get("CFE_HOST", "https://main-cfe-civil-uat.cloud-platform.service.justice.gov.uk") + "/v6/assessments"
```

Comment:
Is this related to LEP-485? (I'm not sure CLA team should be using Eligibility Platform team's UAT environment at all)